user.go

```diff
@@ -93,6 +107,41 @@ func (s *Server) handlerUser(w http.ResponseWriter, r *http.Request) {
 		AddedMovies:  addedMovies,
 	}
 
+	twitchAuth, err := s.data.GetCfgBool(ConfigTwitchOauthEnabled, DefaultTwitchOauthEnabled)
+	if err != nil {
+		s.doError(http.StatusInternalServerError, "Something went wrong :C", w, r)
+		s.l.Error("Unable to get ConfigTwitchOauthEnabled config value: %v", err)
+		return
+	}
+	data.TwitchOAuthEnabled = twitchAuth
+
+	discordAuth, err := s.data.GetCfgBool(ConfigDiscordOauthEnabled, DefaultDiscordOauthEnabled)
+	if err != nil {
+		s.doError(http.StatusInternalServerError, "Something went wrong :C", w, r)
+		s.l.Error("Unable to get ConfigDiscordOauthEnabled config value: %v", err)
+		return
+	}
+	data.DiscordOAuthEnabled = discordAuth
+
+	patreonAuth, err := s.data.GetCfgBool(ConfigPatreonOauthEnabled, DefaultPatreonOauthEnabled)
+	if err != nil {
+		s.doError(http.StatusInternalServerError, "Something went wrong :C", w, r)
+		s.l.Error("Unable to get ConfigPatreonOauthEnabled config value: %v", err)
+		return
+	}
+	data.PatreonOAuthEnabled = patreonAuth
+
+	data.OAuthEnabled = twitchAuth || discordAuth || patreonAuth
+
+	_, err = user.GetAuthMethod(common.AUTH_LOCAL)
```

Comment:
For this case (opposed to the one in `admin.go`) i would actually oppose your suggestion, since user.GetAuthMethod doesnt actually do DB requests and only "iterates" over the user object itself, hence the only error that CAN be returned is the custom "no auth method found" error.